samples/subsys/usb/hid/src/main.c

```diff
@@ -53,6 +53,8 @@ static const u8_t hid_report_desc[] = {
 	HID_MI_COLLECTION_END,
 };
 
+static u8_t report_1[2] = { REPORT_ID_1, 0x00 };
+
 int debug_cb(struct usb_setup_packet *setup, s32_t *len,
```

Comment:
it is not under this change but can you make all these callbacks static?

Reply:
done